src/widgets/CredentialsBox.tsx

```diff
@@ -10,13 +10,16 @@ export class GitCredentialsForm
   extends Widget
   implements Dialog.IBodyWidget<Git.IAuth>
 {
+  passwordPlaceholder: string;
```

Comment:
Purely styling comment:

Could you place this attribute at the bottom of the class with the other attributes and make it private (with the convention of starting with `_`)?

Reply:
Done. I'm guessing `_` is only for the private attribute, so I didn't change it on the parameter.